runtime/src/weights/frame_system.rs

```diff
@@ -34,41 +35,41 @@ impl<T: frame_system::Config> frame_system::WeightInfo for WeightInfo<T> {
 			.saturating_add((1_000 as Weight).saturating_mul(b as Weight))
 	}
 	fn remark_with_event(b: u32, ) -> Weight {
-		(0 as Weight)
+		(2_653_000 as Weight)
```

Comment:
These are the ones, where I really wonder about the stability...

Reply:
yes, but I'd rather go with the more pessimistic value here....